Source/WebCore/platform/WebCorePersistentCoders.cpp

```diff
@@ -446,8 +450,13 @@ template<> struct Coder<GRefPtr<GByteArray>> {
 
         GRefPtr<GByteArray> byteArray = adoptGRef(g_byte_array_sized_new(*size));
         g_byte_array_set_size(byteArray.get(), *size);
+
+        WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // WPE/GTK ports
+        // FIXME: Can't use WTF::span here because it returns std::span<const uint8_t>
+        // but we need std::span<uint8_t>.
         if (!decoder.decodeFixedLengthData({ byteArray->data, *size }))
```

Comment:
You can use `WTF::spanConstCast`, I think this should do:

```suggestion
        if (!decoder.decodeFixedLengthData(WTF::spanConstCast<uint8_t>(WTF::span(byteArray))))
```

Reply:
Nice.